pyro/contrib/epidemiology/compartmental.py

```diff
@@ -371,7 +374,7 @@ def heuristic():
         logger.info("Running inference...")
         max_tree_depth = options.pop("max_tree_depth", 5)
         full_mass = options.pop("full_mass", self.full_mass)
-        model = self._vectorized_model
+        model = self._relaxed_model if self.relaxed else self._vectorized_model
```

Comment:
is `vectorized_model` still the most appropriate name?

Reply:
Sure, I've renamed to `_quantized_model()`.